content/master/getting-started/provider-aws.md

```diff
@@ -201,7 +201,7 @@ The {{< hover label="xr" line="3">}}apiVersion{{< /hover >}} and
 {{< hover label="xr" line="4">}}kind{{</hover >}} are from the provider's CRDs.
 
 
-The {{< hover label="xr" line="6">}}metadata.name{{< /hover >}} value is the 
+The {{< hover label="xr" line="6">}}metadata.generateName{{< /hover >}} value is the 
```

Comment:
i think this `line="6"` can also be updated to `5` so it's highlighting the right line number on hover